virtual_undercloud.yml

```diff
@@ -0,0 +1,104 @@
+---
+- hosts: localhost
+  tasks:
+    - include_tasks: tasks/copykeys.yml
+      vars:
+        hostname: "localhost"
+        ssh_user: "root"
+
+    - name: clean previous vm
+      shell: |
+         source {{ infrared_dir }}/.venv/bin/activate
+         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True --cleanup True
```

Comment:
So, looks like you are using localhost as the hypervisor. This will mean that I cannot run jetpack from my lapto etc right? I think what we can do is, in the case of baremetal we are excluding the first node as undercloud. We can do something similar in the case of virtualized undercloud and use the first node in isntackenv.json as the hypervisor. Thoughts?

Reply:
Agree. Same opinion.